MFA_FlatGlassIndustry.ipynb

```diff
@@ -146,15 +145,18 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "#Plotting FR and BE flat glass production\n",
+    "# Plotting FR and BE flat glass production\n",
     "fig, ax = plt.subplots()\n",
     "\n",
-    "sns.lineplot(data=BE_FR_FlatGlassProd, markers=True, palette=\"GnBu\")\n",
+    "sns.lineplot(data=BE_FR_FlatGlassProd, ax=ax, markers=True, palette=\"GnBu\")\n",
     "\n",
-    "ax.set_xlim(1945,2015)\n",
+    "ax.set_xlim(1945, 2015)\n",
     "ax.set_ylim(ymin=0)\n",
+    "ax.set_ylabel('Kilotons [kt]')\n",
     "\n",
-    "fig.suptitle('Flat glass production [kt/y]', fontsize = 20)"
+    "fig.suptitle('Flat glass production [kt/y]', fontsize=20)\n",
+    "sns.despine()\n",
+    "plt.show()"
```

Comment:
Bon je t'ai mis ca un peu partout.

sns.despine() => Enleve les spines à droite et en haut. Pour éviter le chartjunk. Mais c'eest une question de gout.

`plt.show()` par contre c'est pas anodin du tout. Si jamais tu fais tourner ton code en dehors de jupyter notebook, a un moment tu dois dire à pyplot (via plt.show, fig.draw, ou autre) qu'il faut arrêter la et te montrer le resultat. 

Sinon si tu fais dans jupyter


In [1]: df.plot() 
In [2] df2.plot()
         plt.show()

dans jupuyter pas de soucis. Si dans un fichier .py, alors tu auras le resultat de df et df2 sur le meme graphe, avec poentiellement df2 qui a pas du tout les meme type de données et donc pas les meme xlim/ylim et tu verras meme pas df.